index.src.html

```diff
@@ -836,10 +839,10 @@ <h4 id="algorithm-create" algorithm>Create a `Credential`</h4>
     7.  Return |p|.
   </ol>
 
-  <h4 id="algorithm-require-mediation" algorithm>Require User Mediation</h4>
+  <h4 id="algorithm-require-mediation" algorithm>Prevent Silent Access</h4>
```

Comment:
Please update the ID as well, to something like `algorithm-prevent-silent-access`.